day_two_guide/topics/restore_etcd.adoc

```diff
@@ -0,0 +1,382 @@
+
+*Fix the `peerURLS` parameter*
+
+After restoring the data and creating a new cluster, the `peerURLs` parameter
+shows `localhost` instead the IP where etcd is listening for peer
+communication:
+
+----
+# etcdctl2 member list
+5ee217d17301: name=master-0.example.com peerURLs=http://*localhost*:2380 clientURLs=https://192.168.55.8:2379 isLeader=true
+----
+
+[discrete]
+== Procedure
+
+. Get the member ID from the `etcdctl member list` output.
+
+. Get the IP where etcd is listening for peer communication:
++
+----
+$ ss -l4n | grep 2380
+----
+
+. Update the member information with that IP:
++
+----
+# etcdctl2 member update 5ee217d17301 https://192.168.55.8:2380
+Updated member with ID 5ee217d17301 in cluster
+----
+
+. To verify, check that the IP is in the output of the following:
++
+----
+$ etcdctl2 member list
+5ee217d17301: name=master-0.example.com peerURLs=https://*192.168.55.8*:2380 clientURLs=https://192.168.55.8:2379 isLeader=true
+----
+
+*Add more members*
+
+In the instance joining the cluster:
+
+. Get the etcd name for the instance in the `ETCD_NAME` variable:
++
+----
+# grep ETCD_NAME /etc/etcd/etcd.conf
+----
+
+. Get the IP where etcd listens for peer communication:
++
+----
+# grep ETCD_INITIAL_ADVERTISE_PEER_URLS /etc/etcd/etcd.conf
+----
+
+. Delete the previous etcd data:
++
+----
+# rm -Rf /var/lib/etcd/*
+----
+
+. On the etcd host where etcd is properly running, add the new member:
++
+----
+$ etcdctl2 member add <name> <advertise_peer_urls>
+----
+
+. The command outputs some variables. For example:
++
+----
+ETCD_NAME="master2"
+ETCD_INITIAL_CLUSTER="master1=https://10.0.0.7:2380,master2=https://10.0.0.5:2380"
+ETCD_INITIAL_CLUSTER_STATE="existing"
+----
++
+Add those values to the `/etc/etcd/etcd.conf` file of the new host:
++
+----
+# vi /etc/etc/etcd.conf
+----
+
+. Once those values are replaced, start the etcd service in the node joining
+the cluster:
++
+----
+# systemctl start etcd.service
+----
+
+. Check for error messages:
++
+----
+$ journalctl -fu etcd.service
+----
+
+. Repeat the above for every etcd node joining the cluster.
+
+. Verify the cluster status and cluster health once all the nodes joined:
++
+----
+# etcdctl2 member list
+5cd050b4d701: name=master1 peerURLs=https://10.0.0.7:2380 clientURLs=https://10.0.0.7:2379 isLeader=true
+d0c57659d8990cbd: name=master2 peerURLs=https://10.0.0.5:2380 clientURLs=https://10.0.0.5:2379 isLeader=false
+e4696d637de3eb2d: name=master3 peerURLs=https://10.0.0.6:2380 clientURLs=https://10.0.0.6:2379 isLeader=false
+----
++
+----
+# etcdctl2 cluster-health
+member 5cd050b4d701 is healthy: got healthy result from https://10.0.0.7:2379
+member d0c57659d8990cbd is healthy: got healthy result from https://10.0.0.5:2379
+member e4696d637de3eb2d is healthy: got healthy result from https://10.0.0.6:2379
+cluster is healthy
+----
+
+==== Restoring etcd for v3
+
+The restore procedure for v3 data is similar to the v2 data.
```

Comment:
s/similar to the v2 data./similar to the restore procedure for the v2 data.

Are there restrictions or guidelines for when you restore one, the other, or both types of data?